### DISCUSSION TITLE
ENVS GitHub secret

### DISCUSSION THREAD

**Discussion by Pratyay360** [nikhilbadyal/docker-py-revanced#55] [General]
share your ``ENVS`` secret @nikhilbadyal

**Comment by nikhilbadyal** [comment]
```
PATCH_APPS=youtube,twitter,reddit,youtube_music
BUILD_EXTENDED=True
EXCLUDE_PATCH_YOUTUBE=custom-branding,enable-debugging
EXCLUDE_PATCH_YOUTUBE_EXTENDED=custom-branding-icon-red,custom-branding-icon-blue,custom-branding-icon-revancify,custom-branding-name,custom-package-name,materialyou,hide-my-mix
EXCLUDE_PATCH_YOUTUBE_MUSIC_EXTENDED=custom-branding-music-red,custom-branding-music-revancify,custom-package-name-music,compact-header
ARCHS_TO_BUILD=arm64-v8a
```

**Comment by Pratyay360** [comment]
please make sure to post the ``ENVS`` here if you change it.

**Comment by nikhilbadyal** [comment]
@Pratyay360  They in no way affect the functioning of the script. They just reflect my **personal** preference. You can go through the docs and build one as per your preferences. In any case, I'll try to keep them updated which I already do (in the [export_env.sh](https://github.com/nikhilbadyal/docker-py-revanced/blob/main/export_env.sh) file) but I miss updating those. (sometimes😅)
